ReSwift/CoreTypes/Store.swift

```diff
@@ -134,9 +134,15 @@ open class Store<State: StateType>: StoreType {
     }
 
     open func unsubscribe(_ subscriber: AnyStoreSubscriber) {
+        #if compiler(<5.0)
```

Comment:
In order to correctly support differing compilers, this should be `#if compiler(>=5.0)`, and the if/else contents should be reversed

Reply:
Actually, to go a bit further than this, we'll need this to use an `#if swift(...)` check, since we still support Swift 3.2, which does not have the `compiler` directive. I think I'm going to open discussion elsewhere for dropping Swift 3.2.